service/src/main/java/org/cbioportal/service/impl/VariantCountServiceImpl.java

```diff
@@ -58,8 +58,9 @@ private MolecularProfile validateMolecularProfile(String molecularProfileId) thr
 
         MolecularProfile molecularProfile = molecularProfileService.getMolecularProfile(molecularProfileId);
 
-        if (!molecularProfile.getMolecularAlterationType()
-            .equals(MolecularProfile.MolecularAlterationType.MUTATION_EXTENDED)) {
+        if (!(molecularProfile.getMolecularAlterationType()
+            .equals(MolecularProfile.MolecularAlterationType.MUTATION_EXTENDED) || !molecularProfile.getMolecularAlterationType()
```

Comment:
@ersinciftci im not really sure where the variant count api is used but figured i added this anyway

Reply:
actually there's a double negation here - will update in follow up PR